jetson/ra_kinematics/arm_state.cpp

```diff
@@ -0,0 +1,297 @@
+#include "arm_state.hpp"
+#include "utils.hpp"
+#include <iostream>
+#include <iomanip>
+
+using namespace Eigen;
+using namespace nlohmann;
+using namespace std;
```

Comment:
any way to not use all of std and just add the relevant libraries used from it? for example just `using std::cout`, etc

Reply:
Yeah for sure, we should've done this a while ago